### PR TITLE
New bw

### PR DESCRIPTION
**Pull request by matthiasdemuzere**
- Change BLDFR in Table, to be consistent with mean SO12 ranges
- Remove BW from Table
- Derive new BW, based on BLDFR_URB2D,  FRC_URB2D and SW
- Update JOSS draft accordingly

### CODE REVIEW

**Comment by matthiasdemuzere** [comment]
This change is done now, addressing #26.

File: JOSS/paper.md [hunk 1]
```diff
@@ -82,10 +82,12 @@ Two procedures are followed when assigning the various urban canopy parameters t
 
 **Procedure 1**: **Morphological parameters** are assigned directly to the high-resolution LCZ map, and are afterwards aggregated to the lower-resolution WRF grid. As a result, the method produces a unique urban morphology parameter value for each WRF grid cell. This was found to be more efficient in reproducing urban boundary layer features, especially in the outskirts of the city [@Zonato2020], and is in line with the [WUDAPT-to-COSMO](https://github.com/matthiasdemuzere/WUDAPT-to-COSMO) routine [@Varentsov2020].
 
-Morphological urban canopy parameter values are provided in `LCZ_UCP_default.csv`, and are generally based on values provided in @Stewart2012 and @Stewart2014. In addition:
+Morphological urban canopy parameter values are provided in `LCZ_UCP_default.csv`, and are generally based on values provided in @Stewart2012 and @Stewart2014. Note however that the values of MH_URB2D_MIN, MH_URB2D, MH_URB2D_MAX for LCZ 7 are set to 4, 5 and 6 m instead of 2, 3 and 4 m, because the minimum building height that can assigned to BEP-BEM is 5m if dz_u = 5m (standard value) is used.
```
Comment (theendlessriver13) [inline JOSS/paper.md:85]: typo here - missing **be**? the minimum building height that can **be** assigned

File: w2w/resources/.~lock.LCZ_UCP_lookup.csv# [hunk 2]
```diff
@@ -0,0 +1 @@
+Matthias Demuzere,demuzmp4,rub.geographie.ruhr-uni-bochum.de,14.01.2022 14:47,file:///home/demuzmp4/.config/libreoffice/4;
```
Comment (theendlessriver13) [inline w2w/resources/.~lock.LCZ_UCP_lookup.csv#:1]: Was this checked in by accident? Let me know, then I can easily remove it without a separate commit.

**Review by theendlessriver13 — COMMENTED** [review]
Can't say much about the logical change, but besides the minor remarks this looks good!

File: JOSS/paper.md (hunk 1 above)
Comment (matthiasdemuzere) [reply JOSS/paper.md:85]: yes, good catch! Please adjust!

File: w2w/resources/.~lock.LCZ_UCP_lookup.csv# (hunk 2 above)
Comment (matthiasdemuzere) [reply w2w/resources/.~lock.LCZ_UCP_lookup.csv#:1]: Ha, yes. Libreoffice must have been open when committing. This file can be removed.

**Review by theendlessriver13 — APPROVED** [review]
Fixed the two things and recreated the pdf.

**Pull request merged**